examples/xdc-example/src/app/xdc.virtualAccount.example.ts

```diff
@@ -1,43 +1,42 @@
-  // https://apidoc.tatum.io/tag/Blockchain-addresses#operation/addressExists
-  const account = await xdcSdk.virtualAccount.depositAddress.checkExists(
-    'xdcae14761e5db0ef472848cc4e4a4480311cb7caec',
-  )
-  console.log(`Account information ${JSON.stringify(account)}`)
-
-  // Create a deposit address for a virtual account
-  // https://apidoc.tatum.io/tag/Blockchain-addresses#operation/generateDepositAddress
-  const address = await xdcSdk.virtualAccount.depositAddress.create('635b92d6f9aeb823db30c43f', 9)
-  console.log(`Address information ${JSON.stringify(address)}`)
-
-  // Create multiple deposit addresses for a virtual account
-  // https://apidoc.tatum.io/tag/Blockchain-addresses#operation/getAllDepositAddresses
-  const addresses = await xdcSdk.virtualAccount.depositAddress.createMultiple({
-    addresses: [
-      {
-        accountId: '635b92d6f9aeb823db30c43f',
-        derivationKey: 10,
-      },
-      {
-        accountId: '635b92d6f9aeb823db30c43f',
-        derivationKey: 11,
-      },
-    ],
+  // if you don't already have a wallet, address and private key - generate them
+  // https://apidoc.tatum.io/tag/XinFin#operation/XdcGenerateWallet
+  const { mnemonic, xpub } = await xdcSDK.wallet.generateWallet()
+
+  // https://apidoc.tatum.io/tag/XinFin#operation/XdcGenerateAddressPrivateKey
+  const privateKey = await xdcSDK.wallet.generatePrivateKeyFromMnemonic(mnemonic, 0)
+  // https://apidoc.tatum.io/tag/XinFin#operation/XdcGenerateAddress
+  const to = xdcSDK.wallet.generateAddressFromXPub(xpub, 1)
+
+  // Generate new virtual account for XDC with specific blockchain address
+  // https://apidoc.tatum.io/tag/Account#operation/createAccount
+  const virtualAccount = await xdcSDK.ledger.account.create({
+    currency: 'XDC',
+    xpub: xpub,
   })
-  console.log(`Information for all addresses ${JSON.stringify(addresses)}`)
+  console.log(`Virtual account`, virtualAccount)
+
+  // create deposit address for a virtual account
+  // https://apidoc.tatum.io/tag/Blockchain-addresses#operation/generateDepositAddress
+  const depositAddress = await xdcSDK.virtualAccount.depositAddress.create(virtualAccount.id)
+
+  console.log(`Deposit address`, depositAddress.address)
 
-  // Get all deposit addresses for a virtual account
-  // https://apidoc.tatum.io/tag/Blockchain-addresses#operation/getAllDepositAddresses
-  const addressByAccount = await xdcSdk.virtualAccount.depositAddress.getByAccount('635b92d6f9aeb823db30c43f')
-  console.log(`Information for all addresses ${JSON.stringify(addressByAccount)}`)
+  // FUND YOUR ACCOUNT WITH XDC FROM https://faucet.apothem.network/
+  console.log(`Fund me ${depositAddress.address} to send offchain transaction!`)
+
+  // I wanna send assets from virtualAccount to blockchain address
+  // https://apidoc.tatum.io/tag/Blockchain-operations#operation/XdcTransfer
+  const result = await xdcSDK.virtualAccount.send({
+    senderAccountId: virtualAccount.id,
+    amount: '1',
+    privateKey,
```

Comment:
instead of private key, we should use mnemonic + depositAddress.derivationKey. you are generating private key for address on index 0, but deposit address is generated on index 1 - 0 is always skipped